src/main/java/com/budgetapp/exceptions/severConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.budgetapp.exceptions;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+public class severConfig implements WebMvcConfigurer {
```

Comment:
please pay attention to your naming, you are making these mistakes consistently

```suggestion
public class ServerConfig implements WebMvcConfigurer {
```